flink-python/pyflink/table/table_environment.py

```diff
@@ -49,6 +49,10 @@ class TableEnvironment(object):
 
     def __init__(self, j_tenv, serializer=PickleSerializer()):
         self._j_tenv = j_tenv
+        j_planner_class = j_tenv.getPlanner().getClass()
```

Comment:
The j_tenv does not contains the `getPlanner()` method if it is a batch env under flink planner. We may need to take this into consideration.